website/docs/r/container_app_job.html.markdown

```diff
@@ -0,0 +1,426 @@
+
+* `termination_grace_period_seconds` -  The time in seconds after the container is sent the termination signal before the process if forcibly killed.
+
+---
+
+A `header` block supports the following:
+
+* `name` - (Required) The HTTP Header Name.
+
+* `value` - (Required) The HTTP Header value.
+
+---
+
+A `volume_mounts` block supports the following:
+
+* `name` - (Required) The name of the volume to mount. This must match the name of a volume defined in the `volume` block.
+
+* `path` - (Required) The path within the container at which the volume should be mounted. Must not contain `:`.
+
+---
+
+A `volume` block supports the following:
+
+* `name` - (Optional) The name of the volume.
+
+* `storage_type` - (Optional) The type of storage to use for the volume. Possible values are `AzureFile`, `EmptyDir` and `Secret`.
+
+* `storage_name` - (Optional) The name of the storage to use for the volume.
+
+---
+
+A `secrets` block supports the following:
+
+* `name` - (required) Name of the secret.
+
+* `value` - (required) Value of the secret.
+
+---
+
+A `registries` block supports the following:
+
+* `identity` - (Optional) A Managed Identity to use to authenticate with Azure Container Registry.
+
+* `username` - (Optional) The username to use to authenticate with Azure Container Registry.
+
+* `password_secret_name` - (Optional) The name of the Secret that contains the registry login password.
+
+* `server` - (Optional) The URL of the Azure Container Registry server.
+
+---
+
+A `manual_trigger_config` block supports the following:
+
+* `parallelism` - (Optional) Number of parallel replicas of a job that can run at a given time.
+
+* `replica_completion_count` - (Optional) Minimum number of successful replica completions before overall job completion.
+
+---
+
+A `event_trigger_config` block supports the following:
+
+* `parallelism` - (Optional) Number of parallel replicas of a job that can run at a given time.
+
+* `replica_completion_count` - (Optional) Minimum number of successful replica completions before overall job completion.
+
+* `scale` - (Optional) A `scale` block as defined below.
+
+---
+
+A `schedule_trigger_config` block supports the following:
+
+* `cron_expression` - (Required) Cron formatted repeating schedule of a Cron Job.
+
+* `parallelism` - (Optional) Number of parallel replicas of a job that can run at a given time.
+
+* `replica_completion_count` - (Optional) Minimum number of successful replica completions before overall job completion.
+
+---
+
+A `scale` block supports the following:
+
+* `max_executions` - (Optional) Maximum number of job executions that are created for a trigger.
+
+* `min_executions` - (Optional) Minimum number of job executions that are created for a trigger.
+
+* `polling_interval_in_seconds` - (Optional) Interval to check each event source in seconds.
+
+* `rules` - (Optional) A `rules` block as defined below.
+
+---
+
+A `rules` block supports the following:
```

Comment:
This block seems to do the same job as [`custom_scale_rule`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/container_app#custom_scale_rule) on regular container apps. Unclear why there is a new syntax for container app jobs